src/NeoSharp.Core/Persistence/IRepository.cs

```diff
@@ -54,5 +55,14 @@ public interface IRepository
         /// <returns>Transaction with the specified id / hash</returns>
         Task<Transaction> GetTransaction(UInt256 hash);
         #endregion
+
+        #region Index
+        Task<uint> GetIndexHeight();
+        Task SetIndexHeight(uint height);
```

Comment:
Please remove `SetTotalBlockHeight` and `GetTotalBlockHeight` if is not used now